--- a/src/mVMC/CMakeLists.txt
+++ b/src/mVMC/CMakeLists.txt
@@ -21,24 +21,24 @@
 
 link_directories(${CMAKE_CURRENT_SOURCE_DIR}/../pfapack)
 
-add_executable(vmcdry.out vmcdry.c)
-target_link_libraries(vmcdry.out StdFace m)
-add_executable(vmc.out ${SOURCES_vmcmain} ${SOURCES_sfmt})
-target_link_libraries(vmc.out StdFace)
-target_link_libraries(vmc.out pfapack StdFace ${LAPACK_LIBRARIES} m)
+add_executable(vmcdry vmcdry.c)
+target_link_libraries(vmcdry StdFace m)
+add_executable(vmc ${SOURCES_vmcmain} ${SOURCES_sfmt})
+target_link_libraries(vmc StdFace)
+target_link_libraries(vmc pfapack StdFace ${LAPACK_LIBRARIES} m)
 
 if(USE_SCALAPACK)
   string(REGEX REPLACE "-L[ ]+" "-L" sc_libs ${SCALAPACK_LIBRARIES})
   string(REGEX REPLACE "[ ]+" ";" sc_libs ${sc_libs})
   foreach(sc_lib IN LISTS sc_libs)
-    target_link_libraries(vmc.out ${sc_lib})
+    target_link_libraries(vmc ${sc_lib})
   endforeach(sc_lib)
   message(STATUS "SCALAPACK_LIBRARIES: ${SCALAPACK_LIBRARIES}")
 endif(USE_SCALAPACK)
 
 if(MPI_FOUND)
-  target_link_libraries(vmc.out ${MPI_C_LIBRARIES})
+  target_link_libraries(vmc ${MPI_C_LIBRARIES})
 endif(MPI_FOUND)
-install(TARGETS vmcdry.out RUNTIME DESTINATION bin)
-install(TARGETS vmc.out RUNTIME DESTINATION bin)
+install(TARGETS vmcdry RUNTIME DESTINATION bin)
+install(TARGETS vmc RUNTIME DESTINATION bin)
 add_definitions(-D_mVMC)
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,7 +13,7 @@
 endif(NOT CMAKE_BUILD_TYPE)
 message(STATUS "Build type: " ${CMAKE_BUILD_TYPE})
 
-option(BUILD_SHARED_LIBS "Build shared libraries" ON)
+option(BUILD_SHARED_LIBS "Build shared libraries" OFF)
 enable_language(C Fortran)
 
 set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
--- a/src/pfapack/CMakeLists.txt
+++ b/src/pfapack/CMakeLists.txt
@@ -8,7 +8,3 @@
 include_directories(${MPI_Fortran_INCLUDE_PATH})
 add_library(pfapack ${SOURCES_PFAPACK})
 target_link_libraries(pfapack ${MPI_Fortran_LIBRARIES} ${LAPACK_LIBRARIES})
-install(TARGETS pfapack
-        ARCHIVE DESTINATION lib
-        LIBRARY DESTINATION lib
-        RUNTIME DESTINATION bin)
--- a/src/ComplexUHF/CMakeLists.txt
+++ b/src/ComplexUHF/CMakeLists.txt
@@ -20,4 +20,4 @@
 
 add_executable(UHF ${SOURCES_UHF} ${SOURCES_sfmt})
 target_link_libraries(UHF ${LAPACK_LIBRARIES} m)
-
+install(TARGETS UHF RUNTIME DESTINATION share/mvmc/tool)
--- a/tool/CMakeLists.txt
+++ b/tool/CMakeLists.txt
@@ -10,13 +10,13 @@
 add_executable(greenr2k greenr2k.F90)
 target_link_libraries(greenr2k key2lower ${LAPACK_LIBRARIES})
 
-install(TARGETS greenr2k RUNTIME DESTINATION bin)
+install(TARGETS greenr2k RUNTIME DESTINATION share/mvmc/tool)
 #
 # Scripts
 #
 configure_file(wout2geom.sh wout2geom.sh COPYONLY)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/wout2geom.sh DESTINATION bin
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/wout2geom.sh DESTINATION share/mvmc/tool
         PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
 configure_file(respack2wan90.py respack2wan90.py COPYONLY)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/respack2wan90.py DESTINATION bin
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/respack2wan90.py DESTINATION share/mvmc/tool
         PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
